_data/glossary/reviewer.yaml

```diff
@@ -0,0 +1,10 @@
+id: reviewer
+name: Reviewer
+tags:
+- community
+short-description: |
+  Able to review code for quality and correctness on some part of the project.
```

Comment:
A person who reviews code for quality and correctness on some part of the project.